### ISSUE TITLE
rabbitmqctl fails with "escript: Premature end of file reached"

### ISSUE THREAD

**Issue by dkaelbling** [AlejandroRivera/embedded-rabbitmq#57]
On my macOS Mojave system using embedded-rabbitmq 1.3.1 to install V3_7_18 attempts to invoke rabbitmqctl fail with "escript: Premature end of file reached".  It looks like the extractor isn't handling hard links?

```
dkaelbling-laptop 109% ls -l rabbitmq_server-3.7.18/escript/
total 2056
-rwxr--r-- 1 daveka staff 1842797 Nov 24 11:22 rabbitmq-diagnostics*
-rwxr--r-- 1 daveka staff       0 Nov 24 11:22 rabbitmq-plugins*
-rwxr--r-- 1 daveka staff       0 Nov 24 11:22 rabbitmqctl*

dkaelbling-laptop 114% tar vtjf $TMPDIR/.embedrabbitmq_download/rabbitmq-server-generic-unix-3.7.18.tar.xz | fgrep escript
drwxr-xr-x  0 root   root        0 Sep 17 00:18 rabbitmq_server-3.7.18/escript/
-rwxr-xr-x  0 root   root  1842797 Sep 17 00:18 rabbitmq_server-3.7.18/escript/rabbitmq-diagnostics
hrwxr-xr-x  0 root   root        0 Sep 17 00:18 rabbitmq_server-3.7.18/escript/rabbitmq-plugins link to rabbitmq_server-3.7.18/escript/rabbitmq-diagnostics
hrwxr-xr-x  0 root   root        0 Sep 17 00:18 rabbitmq_server-3.7.18/escript/rabbitmqctl link to rabbitmq_server-3.7.18/escript/rabbitmq-diagnostics
```

**Comment by AlejandroRivera** [comment]
Thanks for the report! I was able to confirm the same issue with my MacOS High Sierra. I'll look into this as soon as i find the time.

**Comment by AlejandroRivera** [comment]
BTW, as a temp workaround you can try using version 3.7.3 on your Mac.

The issue is that with 3.7.18, the Mac-specific distribution is no longer available and it's now using the generic Linux (which has hard links), while the Mac distribution never had them.

**Comment by AlejandroRivera** [comment]
Released [version 1.3.2](https://github.com/AlejandroRivera/embedded-rabbitmq/releases/tag/v1.3.2)

**Comment by AlejandroRivera** [comment]
The fix in 1.3.2 has a limitation in that it disables ALL env vars. 
Will release 1.3.3 with another approach to allow to disable specific env vars and even pass additional env vars. See https://github.com/AlejandroRivera/embedded-rabbitmq/pull/59 for more info

**Comment by AlejandroRivera** [comment]
Version 1.3.3 is now released.
